Platform-Linux/Hardening/SSH-Config/1-Ansible/copy-ssh-key-enforce-pubkey.yaml

```diff
@@ -0,0 +1,38 @@
+# Usage
+# This script will copy your public key and enforce PubkeyAuthentication by disabling password authentication 
+# and ensuring pubkeyauthentication is enabled
+
+# Instructions
+# Make sure the lookup path to your public key is correct.
+
+---
+- name: Copy SSH keys
+  hosts: all
+  become: true
+
+  tasks:
+    - name: Copy SSH public key
+      authorized_key:
+        user: "alex"
+        key: "{{ lookup('file', '~/.ssh/id_ed25519.pub') }}"
+        state: present
```

Comment:
Same thing of making this more generic, probably read from a file and copy all contents over?